.goreleaser.yaml

```diff
@@ -0,0 +1,55 @@
+# This is an example .goreleaser.yml file with some sensible defaults.
+# Make sure to check the documentation at https://goreleaser.com
+before:
+  hooks:
+    - go mod tidy
+builds:
+  - main: ./cmd/celestia
+    binary: celestia
+    env:
+      # NOTE: goreleaser doesn't fully support CGO natively. If CGO is needed
+      # for any node features, this should be removed and a workaround might
+      # need to be created.
+      # REF: https://goreleaser.com/limitations/cgo/
+      - CGO_ENABLED=0
+      - VersioningPath={{ "github.com/celestiaorg/celestia-node/nodebuilder/node" }}
```

Comment:
`# yamllint disable-line rule:line-length`

Reply:
question for the team, the default line length for yamllint is 80, but we can set it to 120 via a config file. I'm pretty sure your other linters look at line length of 120 as well.

Also we set it to 120 here, but looks like this file doesn't get picked up even though it is a common workflow file. https://github.com/celestiaorg/.github/blob/main/.yamllint.yml

So I can either fix this one line, or add the yamllint config file which will help with future PRs as well. 

lmk what you prefer 👍

Reply:
yamllint file is more preferable 👍